### DISCUSSION TITLE
Sister and Giant: A Young Lady Is Reborn in Another World - SuicaoDex

### DISCUSSION THREAD

**Discussion by giscus** [TNTKien/suicaodex#12] [Manga]
# Sister and Giant: A Young Lady Is Reborn in Another World - SuicaoDex

Hinako, a girl who came from another world, made a vow of sisterhood with a giant called Elis, and together they became known as 'The Smallest Giants'. They both have goals that aligned in tracking down people who were significant to them. For Hinako, it is her 'Onee-sama' who was also transported to this world, and for Elis it's the kin she was separated from. However, when collecting their reward after fulfilling a request at the guild, Hinako caught the eye of a knight who serves the church... 

This is the story of a Young Lady and a Giant, two girls who were born to different races and different worlds, and the quest to find their "loved ones" which binds them together.

https://suicaodex.com/manga/21b848be-1bfe-4f97-b92c-62d319d8a093

**Comment by Rudo4510471** [comment]
bộ này khá hay nha
bản Eng ra chap 20 trên mangadex rồi, mong có nhóm dịch
